libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -2367,6 +2367,9 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
     managerAppName: "XSN",
     ticker: "XSN",
     scheme: "xsn",
+    terminated: {
+      link: "https://support.ledger.com/",
```

Comment:
I'm curious: does this link need to be a `support.ledger.com` account ? This coins has no article in our support website, maybe can we link an other url like this coinmarketcap link that shows the coin has unlisted ?

https://coinmarketcap.com/currencies/stakenet/